crates/orchestrator/src/spawner.rs

```diff
@@ -156,7 +156,8 @@ where
                 .map(|var| (var.name.clone(), var.value.clone())),
         )
         .injected_files(files_to_inject)
-        .created_paths(created_paths);
+        .created_paths(created_paths)
```

Comment:
We can remove this, there is no need to use created_paths now that it's created on send_file

Reply:
We still need this for `native`, we should abstract in some way for both providers in a follow up pr.

Reply:
You are right, we should probably at it here (as the Kubernetes provider): https://github.com/paritytech/zombienet-sdk/blob/2a59e27b36f09d1b5b838c6ad237fd662da55f14/crates/provider/src/native/node.rs#L408